spec.bs

```diff
@@ -280,9 +280,16 @@ To <dfn>look up penultimate redemption</dfn> given an [=/origin=] |issuer| and a
 1. If [=redemptionTimes=][(|issuer|,|topLevel|)] [=map/exists=], let |penultimateRedemption| be the |penultimateRedemption| field of the tuple [=redemptionTimes=][(|issuer|,|topLevel|)].
 1. Return |penultimateRedemption|.
 
-To <dfn>look up the key commitments</dfn> for a given [=/origin=] |issuer| and an [=/origin=] |topLevel|, run the following steps:
+An user agent has <dfn>pstKeyCommitments</dfn>, a [=map=] where the keys are an [=/origin=] |issuer|, and the values are a [=map=] |commitment|. Populating |pstKeyCommitments| is [=implementation-defined=].
```

Comment:
```suggestion
An user agent has <dfn>pstKeyCommitments</dfn>, a [=map=] where the keys are an [=/origin=] |issuer|, and the values are a [=map=] |commitment|. Populating [=pstKeyCommitments=] is [=implementation-defined=].
```

I'll add that while this is technically legitimate to write, it's not really making a great effort in terms of interoperability. If it's really impossible to get any more normative than that we should have at least a small non-normative note below explaining why this is entirely implementation-defined and what user agents usually do.

Reply:
Added a note, not sure if there's a way to mark it non-normative?

Reply:
No need to mark it, `Note:`s are non-normative by design.

So, reading that note, how does the browser know which key commitments to fetch in advance? Couldn't one specify any issuer origin here?

Reply:
You can specify any issuer origin in the API, but only the ones that have registered with the user agent will have key commitments/be available to actually use.

Reply:
Huh, this registration process seems like something implementers should be aware of. Is this already in the spec? Otherwise we should probably file a new issue for it.

Reply:
Filing a new issue seems reasonable. Though I wonder whether that belongs in the spec or somewhere else, since each implementer will want to have their own policy/approach to letting issuers register?

Reply:
That's fine, but it should be explained how a user agent might obtain them.

Reply:
Filed #195.